package.json

```diff
@@ -32,26 +32,27 @@
     "publish:major": "npm version major -m 'Release major %s'",
     "postversion": "npm run build && git push && git push --tags && npm publish --access public",
     "test": "npm run build && ava -v && standard -v",
-    "test:watch": "jest --watch"
+    "test:watch": "jest --watch",
+    "lint": "standard -v"
   },
   "devDependencies": {
-    "@ava/babel": "^1.0.0",
-    "@babel/core": "^7.7.5",
-    "@babel/preset-env": "^7.7.6",
-    "@babel/preset-react": "^7.7.4",
-    "ava": "^3.0.0",
-    "esm": "^3.2.25",
-    "puppeteer": "^2.0.0",
-    "react": "^16.12.0",
-    "react-dom": "^16.12.0",
-    "rollup": "^1.27.9",
-    "rollup-plugin-buble": "^0.19.8",
-    "rollup-plugin-commonjs": "^10.1.0",
-    "rollup-plugin-node-resolve": "^5.2.0",
-    "rollup-plugin-replace": "^2.2.0",
-    "rollup-plugin-serve": "^1.0.1",
-    "rollup-plugin-uglify": "^6.0.3",
-    "standard": "^14.3.1"
+    "@ava/babel": "2.0.0",
+    "@babel/core": "7.15.0",
+    "@babel/preset-env": "7.15.0",
+    "@babel/preset-react": "7.14.5",
```

Comment:
should we bump to latest (v7.15.0) to match the other "@babel/*" deps?

Reply:
This is latest :)